.github/workflows/build-deb-pkg.sh

```diff
@@ -0,0 +1,32 @@
+PKG_ROOT=/tmp/avalanche
+DEBIAN_BASE_DIR=$PKG_ROOT/debian
+AVALANCHE_BUILD_BIN_DIR=$DEBIAN_BASE_DIR/usr/local/bin
+AVALANCHE_LIB_DIR=$DEBIAN_BASE_DIR/usr/local/lib/avalanche
```

Comment:
Would the final path for the evm plugin be `/usr/local/lib/avalanche/evm`, or `/usr/local/lib/avalanche/plugins/evm`?

I went for `/usr/local/lib/avalanche/plugins/evm` with the Ansible roles in `scripts/ansible`, but I'm open to changing that.

Reply:
The final dir is `/usr/local/lib/avalanche/evm` . I am still pushing though for automatic lookup of the gecko client to find it there though @StephenButtolph . So far we had `/usr/local/lib/gecko/evm`, but after the renaming of the binary I did proactively change it to that.

Reply:
/usr/local/lib/avalanche/**plugins**/evm makes slightly more sense to me. It namespaces things a bit, making it easier to distinguish other potential extension mechanisms or hooks in the future.

Reply:
I am not sure I agree.

The binary itself is somewhere else in `/usr/local/bin/` - we don't have anything other than plugins so far, and `/usr/local/lib/avalanche` is already like the `plugins` folder in some sense. Unless I am missing something about upcoming potential other binaries which could be placed there. 

I am also moving away from my old tendency to want to organize code for things in the future which are not a need today.

Let's see if someone else chimes in and offers an opinion.